solution/index.js

```diff
@@ -0,0 +1,327 @@
+            alert("Please enter value");
+            }
+            else
+            {
+                newTask.textContent= textField;
+                currentSection.querySelector("input").value = ""; 
+                newTask.classList.add("task");
+                newTask.setAttribute("draggable","true");
+                newTask.setAttribute("ondragstart","drag(event)");
+                newTask.setAttribute("id",Math.random());
+                const ul = currentSection.querySelector("ul");
+                ul.prepend(newTask);
+                addTasksToLocal(target,newTask.textContent);
+            }
+        }
+    }
+
+
+//edit tasks functions
+    document.addEventListener("dblclick", editTask)
+
+    function editTask (event)
+    {
+        if (event.target.classList.contains("task"))
+        {
+            const li = event.target;
+            const oldValue = li.textContent;
+            li.setAttribute("contenteditable",true)
+            li.focus(); 
+            
+            li.addEventListener("blur", unFoucus);
+
+            function unFoucus ()
+            { 
+                const newValue = li.textContent;
+                const section = li.closest("section");
+                const buttonId = section.querySelector("button").id;
+                let arrName = section.id;
+
+                if(li.textContent === ""){
+                     alert("you must enter value"); 
+                     li.remove();
+                     let index = data[arrName].indexOf(oldValue);
+                     data[arrName].splice(index,1);
+                }
+                else{
+                    blurTask (buttonId,arrName,oldValue,newValue)
+                }
+                localStorage.setItem("tasks",JSON.stringify(data))
+            }
+        }
+    }
+    function blurTask (buttonId,arrName,oldValue,newValue)
+    {
+        if ((buttonId === "submit-add-to-do")||
+            (buttonId === "submit-add-in-progress")||
+            (buttonId === "submit-add-done"))
+            {
+                let index = data[arrName].indexOf(oldValue)
+                data[arrName].splice(index,1,newValue)
+        }
+    }
+    
+
+// Search bar function
+function searchSort() {
+    let input = document.getElementById('search').value
+    input=input.toLowerCase();
+    let tasksArr = document.getElementsByClassName('task');
+
+    for (i = 0; i < tasksArr.length; i++)
+     { 
+        if (!tasksArr[i].innerHTML.toLowerCase().includes(input)) {
+            tasksArr[i].style.display="none";
+        }
+        else {
+            tasksArr[i].style.display="list-item";               
+        }
+    }
+}
+const searchbar = document.getElementById("search");
+searchbar.addEventListener("keyup",searchSort);
+
+
+
+//Alt+1/2/3 function
+document.querySelectorAll('.task').forEach(item => {
+    item.addEventListener('mouseover', (e)=>switchTasks(e))  })
+
+let specificLi;
+
+function switchTasks(e)
+{
+    specificLi=e.target;
```

Comment:
That is not a good variable name.
But it doesn't matter, because you shouldn't use global variables anyway. Find another way to achieve what you want.